pom.xml

```diff
@@ -49,6 +49,7 @@
 
     <!-- https://spring.io/projects/spring-boot -->
     <spring-boot.version>2.3.6.RELEASE</spring-boot.version>
+    <spring-boot-web-starter.version>2.4.1</spring-boot-web-starter.version>
```

Comment:
@kilbphilippSAP is this compatible to the other spring libraries we use?

Reply:
@hilmarf will investigate.